### PR TITLE
Bug 1802035: openstack UPI: Document Ansible

### PR DESCRIPTION
**Pull request by pierreprinetti**
Detail a working Ansible UPI workflow in the Openshift Openstack user documentation.

I have changed the order of the steps in order to group all the preparation work at the beginning. That way, the user can then run all playbooks in sequence, without having to go back-and-forth between Ansible and local editing.

Implements OSASINFRA-1108

### CODE REVIEW

File: upi/openstack/README.md [hunk 1]
```diff
@@ -1,43 +1,14 @@
...
-
-The included `requirements.txt` helps using `pip` for gathering the required dependencies in a Python virtual environment:
-
-```shell
-python3 -m venv venv
-source venv/bin/activate
-pip install -U pip
-pip install -r requirements.txt
-```
+This directory contains the Ansible scripts expected to automate most of the command-line work in the [User-Provided-Infrastructure installation](../../docs/user/openstack/install_upi.md).
```
Comment (iamemilio) [inline upi/openstack/README.md]:
```suggestion
This directory contains Ansible that automate the command-line work in the [User-Provided-Infrastructure installation](../../docs/user/openstack/install_upi.md).
```

File: upi/openstack/README.md [hunk 2]
```diff
@@ -1,43 +1,14 @@
...
 
 ## How to use
 
 Customize the cluster properties in the [Inventory](./inventory.yaml) file.
 
 **NOTE:** To deploy with Kuryr SDN, update the `os_networking_type` field to `Kuryr`.
 
-The playbooks are designed to reproduce an installation equivalent to IPI. Customize them as needed. It is advised to customize the teardown playbooks symmetrically.
+The playbooks in this directory are designed to reproduce an IPI installation (that is, the opinionated installation achieved with the `openshift-install` binary). The reason why they are here is that they might be easier to customise. Depending on the change, it is advised to change the teardown playbooks (`down-*`) accordingly.
```
Comment (iamemilio) [inline upi/openstack/README.md]:
```suggestion
The playbooks in this directory are designed to reproduce an IPI installation, but are highly customizable. Please be aware of changes made to the install playbooks that may require changes to the teardown playbooks.
```
Just to simplify things a little

**Comment by pierreprinetti** [comment]
Thanks @iamemilio!

**Comment by pierreprinetti** [comment]
@mandre , you mentioned you wanted to have a look. Thanks to @iamemilio 's generosity, the honor of the /approve label is yours

**Comment by pierreprinetti** [comment]
/hold

> Error: [ERR]: Error building changeset: InvalidChangeBatch: [Tried to create resource record set [name='api.ci-op-pkijdyy4-55b90.origin-ci-int-aws.dev.rhcloud.com.', type='A'] but it already exists]

**Pull request merged**